src/commands/lambda/instrument.ts

```diff
@@ -307,6 +348,7 @@ export const sentenceMatchesRegEx = (sentence: string, regex: RegExp) => sentenc
 
 InstrumentCommand.addPath('lambda', 'instrument')
 InstrumentCommand.addOption('functions', Command.Array('-f,--function'))
+InstrumentCommand.addOption('regExPattern', Command.String('-fR,--functions-regex'))
```

Comment:
Don't forget to update the lambda command documentation as well here https://github.com/DataDog/datadog-ci/blob/master/src/commands/lambda/README.md